server/controllers/artistsController.js

```diff
@@ -58,7 +75,7 @@ const getArtists = asyncHandler(async (req, res, next) => {
 //! This function is might create the wrong artist, if the artist's data is not unique
 const createArtist = asyncHandler(async (req, res, next) => {
   const newArtistObject = createObjectFromQuery(req.body);
-
+  logger.info(`createArtist with artist details: ${newArtistObject}`);
```

Comment:
did we just created artist? or created object from query?
describe what you are doing, be specific, not general (general only in the begining)